pkg/build/pipelines/ruby/require.yaml

```diff
@@ -0,0 +1,85 @@
+name: Test a Ruby package require, with optional load clause
+
+needs:
+  packages:
+    - ${{inputs.ruby}}
+
+inputs:
+  ruby:
+    description: Which Ruby to use
+    default: ruby
+  require:
+    description: |
+      The package to require.
+    required: false
+  requires:
+    description: |
+      Commands to require packages, each line is a separate command. Example:
+        require 'foo'
+        # test that otherthing can be required from asdf
+        require 'asdf'
+        require 'bark' # this is like woof
+
+      full-line and inline comments are supported via '#'
+    required: false
+
+pipeline:
+  - runs: |
+      set +x
+      RUBY="${{inputs.ruby}}"
+      SINGLE_REQUIRE="${{inputs.require}}"
+      MULTIPLE_REQUIRES="${{inputs.requires}}"
+
+      perform_require() {
+        command="$1"
+        if $RUBY -e "$command"; then
+          echo "$RUBY -e \"$command\": PASS"
+        else
+          echo "$RUBY -e \"$command\": FAIL"
+          return 1
+        fi
+      }
+
+      if [ "$RUBY" = "ruby" ]; then
```

Comment:
I think the default 'DEFAULT' or some other non-usable string is prefereable here.
The caller could very legitmately have `ruby` in the PATH and want that to be tested.  At least I can't think of a good reason to "fix" it for them.

Did you have one?